core/templates/dev/head/components/private_info_tooltip/PrivateInfoTooltipDirective.js

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the private information warning tooltip.
+ */
+
+oppia.directive('privateInfoTooltip', [
```

Comment:
have you imported this .js file where it's required?

Reply:
No, I've just used this directive in `text_input_interaction_directive.js`.

Reply:
So you have to import this where it's required (like [this](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/library/library.html#L222) but in a suitable place).

Reply:
Thanks, done!